BUILDING.md

```diff
@@ -40,9 +40,14 @@ sudo unzip protoc-3.11.4-linux-x86_64.zip -d /usr/local
 `containerd` uses [Btrfs](https://en.wikipedia.org/wiki/Btrfs) it means that you
 need to satisfy these dependencies in your system:
 
-* CentOS/Fedora: `yum install btrfs-progs-devel`
+* CentOS 7 / Fedora: `yum install btrfs-progs-devel`
+  * Note with CentOS 9:  BTRFS has been deprecated in RHEL / CentOS 7.4, and removed in RHEL/CentOS 9 .
+    [Btrfs has been deprecated](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/ch-btrfs).[release_notes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.4_release_notes/chap-red_hat_enterprise_linux-7.4_release_notes-deprecated_functionality_in_rhel7#idm139789147351408)
 * Debian/Ubuntu: `apt-get install btrfs-progs libbtrfs-dev`
   * Debian(before Buster)/Ubuntu(before 19.10): `apt-get install btrfs-tools`
+* For unsupported [Btrfs](https://en.wikipedia.org/wiki/Btrfs) system:
+  * We suggest that using the no_btrfs build-flag to build without btrfs support.
+   
```

Comment:
There's some trailing whitespace here, which is disallowed in the project checks:

> FAIL - has whitespace errors. See `git show --check 189384ca68730834ecbe09b59c56497a552070de`.

```suggestion
```

Reply:
thanks for review.
@samuelkarp